card/lib/card/migration.rb

```diff
@@ -42,8 +42,10 @@ def schema_mode mig_type=type
       original_suffix = ActiveRecord::Base.table_name_suffix
 
       ActiveRecord::Base.table_name_suffix = new_suffix
+      ActiveRecord::SchemaMigration.reset_table_name
       yield
       ActiveRecord::Base.table_name_suffix = original_suffix
+      ActiveRecord::SchemaMigration.reset_table_name
     end
```

Comment:
Actually, it's three different tables: schema_migrations, schema_migrations_core_cards and schema_migrations_deck_cards. I thought about adding models for our extra tables but Rails doesn't seem to support this in any way. The `reset_table_name` is the much simpler approach.